src/game/server/instagib/gamelogic.cpp

```diff
@@ -300,9 +300,9 @@ void CGameContext::InstagibUnstackChatMessage(char *pUnstacked, const char *pMes
 
 int CGameContext::GetDDNetInstaWeapon()
 {
-	if(str_comp_nocase(g_Config.m_SvGametype, "gctf") == 0)
+	if(str_comp_nocase(g_Config.m_SvGametype, "ictf") == 0 || str_comp_nocase(g_Config.m_SvGametype, "gdm") == 0 || str_comp_nocase(g_Config.m_SvGametype, "gtdm") == 0)
 		return WEAPON_GRENADE;
-	if(str_comp_nocase(g_Config.m_SvGametype, "ictf") == 0)
+	if(str_comp_nocase(g_Config.m_SvGametype, "ictf") == 0 || str_comp_nocase(g_Config.m_SvGametype, "idm") == 0 || str_comp_nocase(g_Config.m_SvGametype, "itdm") == 0)
```

Comment:
Could be this fancy wanna be smart ass condition instead
```g_Config.m_SvGametype[0] == 'g'```
```g_Config.m_SvGametype[0] == 'i'```

It would be harder to reason about. It would be harder to search for in the code.
It would be more performant. But thats not relevant since the string compare is fast.
It would be more flexible. But do we even want to add more iXXX and gXXX modes?

Now that I think about the last one I like the idea and opened a [discussion issue](https://github.com/ZillyInsta/ddnet-insta/issues/97) for that. Anyways this is not blocking the pull request from my side either version can be merged.